qutip/settings.py

```diff
@@ -48,6 +48,7 @@
 # number of cpus (set at qutip import)
 num_cpus = 0
 # flag indicating if fortran module is installed
```

Comment:
We used to have a Fortran version of the MC solver -- long since dead. So we could remove this if we wanted, but no real need to.